app/scripts/arbeitsangebote/overview/arbeitsangeboteoverview.html

```diff
@@ -60,6 +60,6 @@ <h3  class="panel-title"><span class="navbar-brand" translate>Arbeitsangebote</s
     <oo-email-dialog on-close="closeCreateEMailDialog" url="url" message="message" ids="kundeIdsMailing" entity="entity"></oo-email-dialog>
   </oo-sliding-detail>
   <oo-sliding-detail slide-in="showGenerateReport" slide-out-function="closeBerichtFunct()">
-    <oo-generate-report on-close="closeBericht" projekt-vorlagen="projektVorlagen()" on-generated="closeBericht" post-path="'kunden/berichte/arbeitsangebot'" ids="checkboxes.ids" default-file-name="'Arbeitsangebot.odt'|translate"></oo-generate-report>
+    <oo-generate-report on-close="closeBericht" projekt-vorlagen="projektVorlagen()" on-generated="closeBericht" post-path="'kunden/berichte/arbeitsangebot'" ids="checkboxes.ids" pdf_generation_options="'false'" default-file-name="'Arbeitsangebot.odt'|translate"></oo-generate-report>
```

Comment:
(sorry, didn't see that on the core modification:)

The attributes on the directives are all with `-` instead of `_` . I suggest to stick to this pattern. And must the `false` be quoted? So it would be:

`pdf-generation-options="false"`

Reply:
Ah, I even see you did it this way for most of the others.